test/unit/models/profileDiffs.test.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * This eslint rule is disabled because of https://github.com/nodesecurity/eslint-plugin-security/issues/21
+ * It gives linting errors in testing the DB data with keys from fixtures
+ */
+/* eslint-disable security/detect-object-injection */
```

Comment:
Please solve the issues, find the reason why its giving lint error and try solving it

Reply:
Ohh alright. Will resolve this.

Reply:
@prakashchoudhary07 Resolved. No linting issue now.